fabric-tool-attribute-api-v1/src/main/java/net/fabricmc/fabric/api/tool/attribute/v1/ToolManager.java

```diff
@@ -0,0 +1,43 @@
+package net.fabricmc.fabric.api.tool.attribute.v1;
+
+import net.minecraft.block.BlockState;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tag.Tag;
+import net.minecraft.util.ActionResult;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.util.TriState;
+import net.fabricmc.fabric.impl.tool.attribute.ToolManagerImpl;
+
+public final class ToolManager {
```

Comment:
Now that these are publicly exposed, they're *definitely* going to need javadocs.